frontend/src/components/NewRunParameters.test.tsx

```diff
@@ -37,6 +37,20 @@ describe('NewRunParameters', () => {
     expect(shallow(<NewRunParameters {...props} />)).toMatchSnapshot();
   });
 
+  it('clicking the open editor button for json parameters display an editor', () => {
+    const handleParamChange = jest.fn();
+    const props = {
+      handleParamChange,
+      initialParams: [{ name: 'testParam', value: '{"test":"value"}' }],
+      titleMessage: 'Specify json parameters required by the pipeline',
+    } as NewRunParametersProps;
+    const tree = mount(<NewRunParameters {...props} />);
+    tree.find('Button#toggleEditorBtn').simulate('click');
```

Comment:
Hey, sorry I missed a problem.
id should be unique in the web page, but there can be multiple editor buttons.

This id is only for testing purpose, a widely accepted convention is to use attribute `data-testid` as an identifier if a better selector cannot be found.

Reply:
Recommend finding the button by its text:
https://github.com/kubeflow/pipelines/blob/e824c0b2f63a8891a18f437a1ad2b94990dd67a9/frontend/src/components/viewers/Tensorboard.test.tsx#L147
the hostNodes command filters to only html nodes.

(that's also how users find a button)